src/components/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,26 @@
+import "./sidebar.css";
+import { Link } from "react-router-dom";
+export const SideBar = () => {
+  return (
+    <ul className="sidebar-container list-style-none">
```

Comment:
@deekshithmd  here <link> element is repetitive so you can create one component for it and for path you can use array and you can iterate over array to give respective paths.
correct me if what i'm saying if not possible!!

Reply:
I will check, Thank you for suggesting

Reply:
It is possible by creating object of item data like pagename, icon class and path.